python/cli/prelude_cli/views/iam.py

```diff
@@ -25,6 +25,18 @@ def register_account(controller):
     print_json(data=creds)
 
 
+@iam.command('update-account')
+@click.option('-m', '--mode',
+              help='provide a mode',
+              default='manual', show_default=True,
+              type=click.Choice(['manual', 'frozen'], case_sensitive=False))
```

Comment:
if we might add more modes, I'd dynamically create this list from the enum like we do elsewhere

Reply:
Agreed